app/models/music.rb

```diff
@@ -3,7 +3,7 @@ class Music < ApplicationRecord
   belongs_to :exercise
 
   validates :bpm, presence: true, numericality: { greater_than: 10, lower_than: 500 }
-  validates :key_signature
+  validates :key_signature, presence: true
   validates :mode, presence: true
   validates :notes, presence: true
   validates :chords, presence: true
```

Comment:
Might be better to refactor this to mostly single line validations 

https://guides.rubyonrails.org/active_record_validations.html#presence

Reply:
If you refer to the :key_signature, it is a single line: the red one was deleted

Reply:
Did you see the link?

You can do `validates :key_signature, :mode, :notes, :chords, :note_values, :chord_values, presence: true`

It keeps it DRY